mypy/types.py

```diff
@@ -1189,6 +1189,29 @@ def deserialize(cls, data: JsonDict) -> 'TypeType':
         return TypeType(Type.deserialize(data['item']))
 
 
+class ClassVarType(Type):
```

Comment:
As I already mentioned, I think this should be rather a flag in ``Type`` base class or in the ``Var`` symbol node (maybe ``is_classvar`` flag in some analogy to ``is_classmethod``).